### PR TITLE
Database

### PR DESCRIPTION
**Pull request by Lockeriano**
Migrations done, models created.

### CODE REVIEW

File: test/models/ingridient_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class IngridientTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (goozzik) [inline test/models/ingridient_test.rb]: We don't want to use standard library provided by rails for testing. Please remove `test` dir from repo, we don't need this.

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def index
+  end
+end
```
Comment (goozzik) [inline app/controllers/welcome_controller.rb]: Can you please prepare setup for https://github.com/rspec/rspec-rails which will be our testing library and create basic test for this controller? You should be able to achieve this by simply following their `README.md`.

File: spec/controllers/welcome_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe WelcomeController do
+  context "get index" do
+    it "succesfully show home page" do
+      get :index
```
Comment (goozzik) [inline spec/controllers/welcome_controller_spec.rb]: 1. Let's move our action call to subject at the top

```ruby
subject(:perform_request) { get :index }
```

2. We miss some assertion. Currently you just call the `index` action and check nothing.

```ruby
before { perform_request }

it 'returns success response' do
   expect(response).to be_success
end
```

File: test/application_system_test_case.rb [hunk 4]
```diff
@@ -1,5 +0,0 @@
-require "test_helper"
-
-class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
-end
```
Comment (goozzik) [inline test/application_system_test_case.rb]: Please remove this file.

Comment (Lockeriano) [reply test/application_system_test_case.rb]: This is already removed such as whole test dir.

Comment (goozzik) [reply test/application_system_test_case.rb]: sorry, right

**Pull request merged**